mobile/src/containers/onboarding/BackupPhotosVideos.jsx

```diff
@@ -39,4 +41,4 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
   }
 })
 
-export default connect(mapStateToProps, mapDispatchToProps)(BackupPhotosVideos)
+export default connect(mapStateToProps, mapDispatchToProps)(translate()(BackupPhotosVideos))
```

Comment:
a HOC (High Order Component) is a function that takes a component as a parameter and returns a component.
Given `f()` and `g()`, two HOC, we must have `f(g(x)) === g(f(x))`. But reading [this wikipedia article about function composition](https://en.wikipedia.org/wiki/Function_composition) it may not be commutative.